packages/wonder-blocks-dropdown/src/components/combobox.tsx

```diff
@@ -330,6 +329,26 @@ export default function Combobox({
         [setFocusedIndex],
     );
 
+    /**
+     * Filters the items based on the input value. This is used when the
+     * `autoComplete` prop is set to `list` or `both`.
+     */
+    const filterItems = React.useCallback(
```

Comment:
**note:** Just moved this function next to `focusOnFilteredItem` as both did similar things (not the same things, but pretty similar).